src/commands/generate.ts

```diff
@@ -98,7 +114,7 @@ export default class GenerateCommand extends Command {
         type: 'input',
         validate: (v: string) => (v !== '' ? true : 'required input!'),
       },
-    ]);
+    ] as Questions);
```

Comment:
```suggestion
    ] as inquirer.Question[]);
```

[IMO] I want to remove the [L.99](https://github.com/cats-oss/scaffdog/pull/44/files#diff-9a5cd32547b74e35e455dd0fed11d95dR99) type alias. And use `inquirer.Question[]`  instead.

Reply:
@tsuyoshiwada thx :)
fix it. https://github.com/cats-oss/scaffdog/pull/44/commits/9a034c35a5da80b7ffef1d36cd17ef11aeb6d8f3